todo_app/trello.py

```diff
@@ -0,0 +1,88 @@
+import requests
+import os 
+from dotenv import load_dotenv
+from todo_app.trello_card import TrelloCard
+
+class Trello:
+
+    def __init__(self): 
+        load_dotenv()
+        self.trello_key=os.getenv('API_KEY')
+        self.trello_token=os.getenv('API_TOKEN')
+        self.board_name=os.getenv('BOARD_NAME')
+        self.memberName=os.getenv('MEMBERS_NAME')
+        self.to_do_list_name=os.getenv("TO_DO_LIST_NAME")
+        self.done_list_name=os.getenv("DONE_LIST_NAME")
```

Comment:
The README needs to be updated. It should contain everything the reader needs to know to get the app up and running, i.e. the app's dependency on Trello and these env vars.

You could also add these to .env.template with placeholder/empty values to make setup easier.

Reply:
done